README.md

```diff
@@ -366,6 +366,9 @@ If you cannot find suitable test data on this repository, please contact us on t
   * 'test.tsv': exemplary tab-separated file obtained from [here](https://bioinf.shenwei.me/csvtk/usage/#split)
 * txt
   * 'hello.txt': one-line txt file
+  * 'ktimporttext.txt': [Krona ktimporttext](https://github.com/marbl/Krona) test file
+  * 'kaijiu_out4krona.txt': [Kaiju output for Krona](https://github.com/bioinformatics-centre/kaiju#creating-input-file-for-krona) for [Krona ktimporttext](https://github.com/marbl/Krona) test file
+  * 'kraken2_report.txt': [Kraken2 report file](https://github.com/DerrickWood/kraken2/wiki/Manual) for [Krona ktimporttext](https://github.com/marbl/Krona) test file
```

Comment:
We dont need thus one as a Kraken report already exists: https://github.com/nf-core/test-datasets/blob/modules/data/genomics/sarscov2/metagenome/test_1.kraken2.report.txt